src/features/import/hooks/useColumn.ts

```diff
@@ -56,37 +56,46 @@ export default function useColumn(orgId: number) {
     return !!exists;
   };
 
-  const columnOptions: Option[] = [];
+  const columnOptionsIDPartition: Option[] = [];
```

Comment:
Move this hard-coded array into `MappingRow` (see previous comment for reasoning).